`make lint-monitoring` fails on release-1.14

<!-- Make sure that you visit our User Guide at https://kubevirt.io/user-guide.
-->

**What happened**:
I have the following error while running `make lint-monitoring`:
```
10:42 $ make lint-monitoring 
go install github.com/kubevirt/monitoring/monitoringlinter/cmd/monitoringlinter@e2be790
monitoringlinter ./...
monitoringlinter: internal error: package "strings" without types was imported from "github.com/kubevirt/hyperconverged-cluster-operator/pkg/patch"
make: *** [Makefile:273: lint-monitoring] Error 1

```
I am using Go 1.24.2


**What you expected to happen**:
It should work regardless the version of Go that is installed on the developer's host.

**How to reproduce it (as minimally and precisely as possible)**:
Steps to reproduce the behavior.
1. Install Go .1.24.2
2. Git checkout release-1.14
3. Make sanity / make linit-monitoring

--- a/Makefile
+++ b/Makefile
@@ -269,7 +269,7 @@ lint-metrics:
 	./hack/prom_metric_linter.sh --operator-name="kubevirt" --sub-operator-name="hco"
 
 lint-monitoring:
-	go install github.com/kubevirt/monitoring/monitoringlinter/cmd/monitoringlinter@e2be790
+	go install github.com/kubevirt/monitoring/monitoringlinter/cmd/monitoringlinter@v0.0.10
 	monitoringlinter ./...
 
 bump-hco:

